src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -47,6 +47,7 @@ export class ArtworkSidebarBidAction extends React.Component<
 
   @track((props: ArtworkSidebarBidActionProps) => ({
     artwork_slug: props.artwork.id,
+    product_id: props.artwork.id,
```

Comment:
I have an issue here in that the `ArtworkSidebarBidAction_artwork` representation only has a slug but no Mongo ID declared on it. How do I regenerate the graphql definition for the Artwork here? I need to pull in `_id` on the artwork.